src/ray/protobuf/common.proto

```diff
@@ -116,40 +116,42 @@ enum ErrorType {
   // but this object is not reconstructable (created by actor, ray.put, or a
   // borowed ObjectRef).
   OBJECT_UNRECONSTRUCTABLE = 2;
+  // Indicates that a task failed because the function or class couldn't be loaded.
+  FUNCTION_LOADING_ERROR = 3;
```

Comment:
Could we append `FUNCTION_LOADING_ERROR` to the last one, 15? The change here cause the following invalid modifications.